_together/02-findings.md

```diff
@@ -10,6 +10,7 @@ slug: findings
 hero_title: Findings
 hero_subtitle: Teams are thinking about how to create digital experiences that embrace all users and are effective, accessible, and respectful.
 chapter: true
+excerpt: ""
```

Comment:
Removes excerpts that we don't use anyway for this page, since they were throwing Jekyll warnings